content/02.introduction.md

```diff
@@ -127,6 +127,20 @@ After a virus enters into a host cell, its antigen is presented by major histoco
 One of the main immune responses that contribute to the onset of Acute Respiratory Distress Syndrome (ARDS) in COVID-19 patients is the cytokine storm, which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.
 In addition to respiratory distress, this mechanism leads to organ failure and death in severe COVID-19 cases [@doi:10.1016/j.jpha.2020.03.001].
 
+There are several ways in which the SARS-CoV virus can evade the human immune response.
+One way is through CD163+ and CD68+ macrophage cells which are crucial for the establishment of SARS in the body [@doi:10.1038/mi.2015.127].
+These cells most likely serve as viral reservoirs that help shield SARS-CoV from the innate immune response.
+According to a study on the viral dissemination of SARS-CoV in Chinese macaques, the expression of CD163 cells was reduced in viral RNA+ cells when monocytes differentiate toward dendritic cells [@doi: 10.1038/mi.2015.127].
+This lack of active viral replication allows SARS-CoV to escape innate immunity because reduced levels of viral RNA allow the virus to avoid both natural killer (NK) cells and Toll-like receptors [@doi: 10.1038/mi.2015.127].
```

Comment:
```suggestion
This lack of active viral replication allows SARS-CoV to escape innate immunity because reduced levels of viral RNA allow the virus to avoid both natural killer (NK) cells and Toll-like receptors [@doi:10.1038/mi.2015.127].
```